e3/testsuite/__init__.py

```diff
@@ -321,6 +337,90 @@ def dump_testsuite_result(self):
         """To be implemented."""
         pass
 
+    def dump_xunit_report(self, filename):
```

Comment:
My be a good thing to have by default the possibility to generat xunit reports. Though I would put the implementation in a separate module (e3.testsuite.report.xunit ?)

Reply:
Ok